### ISSUE TITLE
Fistule does not spawn (for atma of VV)

### ISSUE THREAD

**Issue by benfolka** [LegionDark/Issues#753]
**Date & Time**: 21.06 (gmt +2)

**FFXI Client Version (use `/ver`)**: latest

**Character Name**: Clay

**Nation**: SAndoria

**Job(level)/Sub Job(level)**: 99 war/dnc

**NPC or Monster or item Name**: Fistule

**Zone name & Coordinates (use `@where`)**: Abyssea - Konschtat G-3

**ffxiah.com link (for items issues only)**: 

**Multi-boxing? (multiple clients on same connection)**: 

**Steps To Reproduce / any other info**: Fistule does not spawn

**Comment by gori123** [comment]
this also seems to be a requirement for a lunar abyssite which is important for game.

**Comment by R3P0FFXI** [comment]
@benfolka @gori123 this has a temp workaround and should be in Tuesdays maint per @TeoTwawki

**Comment by TeoTwawki** [comment]
In trunk for next maint I:
* Fixed his spawn, was a bad family ID
* I have made the mob target-able normally, which should have been done ages ago.
* I have added some partial code (commented out till its 100% done) for making him agro and target-able in the retail way by dragging Bloodguzzler or Guimauve near him. For now you guys will be able to just walk up and attack, no special things needed.

Leaving issue open till after maint.
